app/client/components/identity/idapi/user.ts

```diff
@@ -76,7 +76,9 @@ export const isErrorResponse = (error: any): error is UserAPIErrorResponse => {
 const toUserApiRequest = (user: Partial<User>): UserAPIRequest => {
   const { countryCode: countryCode, localNumber: localNumber } = user;
   const telephoneNumber =
-    countryCode && localNumber ? { countryCode, localNumber } : undefined;
+    countryCode && localNumber
+      ? { countryCode, localNumber: `${localNumber}` }
+      : undefined;
```

Comment:
maybe a stupid question but why did we need this change here ?

Reply:
convert localNumber to string before passing it to IDAPI.
The Identity API expects a string - but right now the JSON parser is very lenient so numbers are accepted

Reply:
ah I might be misunderstanding this code then,  is this localNumber not the same localNumber that was redefined to be a string in models.ts?

Reply:
ah sorry, I now see that it was actually changed the other way around, it used to be a string and now it's a number 🤔

Reply:
the form input is a number, then we convert it to a string before sending it to idapi